alternator/executor.cc

```diff
@@ -529,6 +537,11 @@ future<executor::request_return_type> executor::delete_table(client_state& clien
         auto group0_guard = co_await mm.start_group0_operation();
 
         if (!p.local().data_dictionary().has_schema(keyspace_name, table_name)) {
+            // DynamoDB returns validation error even when table does not exist
+            // and the table name is invalid.
+            validate_table_name(table_name);
```

Comment:
I'd say for readability it should be fine to add validate right after table_name variable above, we don't need to optimize for delete table operation

Reply:
Done!